crates/fuel-core/src/combined_database.rs

```diff
@@ -94,6 +94,19 @@ impl CombinedDatabase {
         })
     }
 
+    /// A test-only temporary rocksdb database with given rewind policy.
+    #[cfg(feature = "rocksdb")]
```

Comment:
Done in 5251318e5e

Reply:
I reverted this in beaee56e2e since it seems like we're using it with when empty path is provided, and I'm not sure if we need that outside tests as well.